spec/integration/create_burndown_spec.rb

```diff
@@ -1,65 +1,67 @@
 require_relative 'integration_spec_helper'
 
-include GivenFilesystemSpecHelpers
-include CliTester
+module CreateBurndownSpec
```

Comment:
This is an RSpec test not a module! :see_no_evil:  and the same for the other RSpec files...